doc/API.md

```diff
@@ -311,6 +311,7 @@ Additionally, there are a number or properties that allow configuration of diffe
 | `absolutePaths` | `false` | Use absolute paths when creating element selectors
 | `iframes`       | `true`  | Tell axe to run inside iframes
 | `elementRef`    | `false` | Return element references in addition to the target
+| `restoreScroll` | `false` | Scrolls elements back to before axe started
```

Comment:
@dylanb It looks like it adds +/- 20ms to the execution when using it in dqpl-react. At most the impact is 15%, which is the case for very small components. For larger once this obviously goes down as test time goes up.